### DISCUSSION TITLE
Trip recording - extra steps

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#11092] [Q&A]
Have the Trip recording plugin been updated? On my phone (Android, OsmAnd+) I now suddenly get a new prompt every time I want to start recording a new track that asks me to confirm the logging interval. It used to be that with the shortcut on the map I could easily start a recording in the background with the default settings. The new process is a little annoying since I rarely want to make any changes to the settings but now have to click through a couple of prompts to get back to the map every time.

In addition I can't figure out if there is a way to stop a trip recording without saving it? When I press stop there is a warning that 'All unsaved data will be lost' but the only options are to press 'Save and stop recording' or 'Cancel' (and resume recording).

**Comment by tommycrock** [comment]
It has. And you are right that there's no way to stop recording without saving. See https://github.com/osmandapp/OsmAnd/issues/10924

**Reply by ERYpTION** [reply]
Ok. Hopefully stopping without saving and a less cumbersome recording start option will be added in a future update.

**Reply by thompehw** [reply]
Like you, my "use case" is that I usually want to start a recording quickly and I only want to save it if I know I encountered something interesting (e.g. to map later on OSM). For this (presumably not for many other use-cases) the new design is worse.  But I noticed that when it insists on saving, if you click on the "x" next to the filename then save, you get a toast message saying something like "empty filename" and the track isn't saved.

**Reply by tommycrock** [reply]
@thompehw that sounded like an interesting workaround but when I've just tried it it didn't work for me. I pressed stop, save, X to delete the filename and got the message about empty filename, but when I checked it was still saved (as would be expected - it's already saved).

**Reply by thompehw** [reply]
> @thompehw that sounded like an interesting workaround but 

My mistake - you are correct. 

Looking at the thread on the issues page, it's probably intentional that you can't stop without saving. But this is such a pain if you only occasionally want to save a track because it was interesting! You either have to immediately delete uninteresting tracks (making the number of taps even more ridiculous) or later try to find the interesting one among all the unnecessary recordings!
